batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -34,33 +34,43 @@ public EmployeeController() {
 
   @Override
   protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-    String[] pathParts = getPathParameters(request);
-
-    if (pathParts.length == 3 && AppConstants.CREATE.equals(pathParts[2])) {
+    String page = getPageFromPath(request);
+    switch (page) {
+    case AppConstants.CREATE:
       createProcess(request, response);
-    } else if (pathParts.length == 4 && AppConstants.EDIT.equals(pathParts[3])) {
+      break;
+    case AppConstants.EDIT:
       editProcess(request, response);
-    } else if (pathParts.length == 4 && AppConstants.DELETE.equals(pathParts[3])) {
+      break;
+    case AppConstants.DELETE:
       deleteProcess(request, response);
-    } else {
+      break;
+    default:
       displayPageNotFound(request, response);
+      break;
     }
   }
 
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-    String[] pathParts = getPathParameters(request);
-
-    if (pathParts.length == 2 && AppConstants.EMPLOYEE.equals(pathParts[1])) {
+    String page = getPageFromPath(request);
+    System.out.print(page);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)